docs/querying/api.md

```diff
@@ -653,6 +653,52 @@ curl -G http://localhost:9091/api/v1/targets/metadata \
 }
 ```
 
+## Querying metric metadata
+
+Similarly to the previous endpoint `api/v1/targets/metadata`, it returns metadata about metrics currently scrapped by targets. However, it does not provide any target information.
```

Comment:
Let's not refer to other endpoints and their ordering on this page, as that'll change over time.

Reply:
I wanted to avoid confusion a bit as including this one, we now have three sections that refer to metadata. `Querying metadata`, `Querying target metadata`, and `Querying metric metadata`.

The intention was to hint the reader ahead of time what are some of the differences - I'll make the change.